src/just_auth/core.clj

```diff
@@ -139,13 +143,22 @@
         (f/fail (t/locale [:error :core :not-active])))
       (f/fail (str (t/locale [:error :core :account-not-found]) email))))
 
+  (exists? [_ email]
+    (if-let [account (account/fetch account-store email)]
+      true false))
+
```

Comment:
I really think we should remove this call and the call bellow should return either the account or a failjure for consistency